packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/modal/ReactModalHostManager.kt

```diff
@@ -93,6 +93,12 @@ public class ReactModalHostManager :
   @ReactProp(name = "identifier")
   public override fun setIdentifier(view: ReactModalHostView, value: Int): Unit = Unit
 
+  @ReactProp(name = "testID")
```

Comment:
@ReactProp is already used in parent [BaseViewManager](https://github.com/facebook/react-native/blob/main/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java#L274) so it should not be used here.

Reply:
There is also formatting issue. I'll make some changes and merge this.